stories/CheckboxGroup-stories.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import type { Meta, StoryObj } from '@storybook/react';
+import { forwardRef, useEffect, useRef, useState } from 'react';
+import {
+  type CheckboxGroupProps,
+  CheckboxGroup,
+} from '../src/components/checkbox-group/CheckboxGroup';
+import { toTextControlArgTypes } from './helpers/arg-types';
+
+type CheckboxGroup = typeof CheckboxGroup;
```

Comment:
Using the same name here is disturbing for me. But I guess TypeScript is smart enough to make the difference between the type definition and the actual Component based on the context where it's used.

Reply:
Yeah, that's correct.